### PR TITLE
Fix icons with custom images

### PR DESCRIPTION
**Pull request by joaomsneto**

### CODE REVIEW

File: src/xdgmenumaker [hunk 1]
```diff
@@ -291,9 +291,11 @@ def icon_full_path(icon):
     # in non-standard directories
     if os.path.exists(icon):
         if max_icon_size:
+	    if os.path.splitext(icon)[1] == ".svg":
+            	return icon
             return icon_max_size(icon)
         else:
-            return None
+            return icon
```
Comment (gapan) [inline src/xdgmenumaker]: You're right about this.

File: src/xdgmenumaker [hunk 2]
```diff
@@ -291,9 +291,11 @@ def icon_full_path(icon):
     # in non-standard directories
     if os.path.exists(icon):
         if max_icon_size:
+	    if os.path.splitext(icon)[1] == ".svg":
```
Comment (gapan) [inline src/xdgmenumaker]: You're right that we need to check for svg icons separately. But we need to discard svg icons altogether. I don't think any of the supported WMs can use svg icons! So, I see 2 problems with this:
1. We don't want so show svg icons. It should return None in the next line.
2. The indentation is wrong. You're missing one more space before the if check.

Comment (gapan) [inline src/xdgmenumaker]: Sorry, the problem with the indentation is that you're using a tab character, instead of spaces as with the rest of the file.

Comment (joaomsneto) [inline src/xdgmenumaker]: I'm sorry, I have some problems with spaces and tabs for indentation. Thank you for your answer.

**Comment by gapan** [comment]
Thanks for bringing this to my attention. I've changed the code and dealt with the problem in a slightly different way.

**Pull request closed without merging**